emulator/drivers/sun_sensor.py

```diff
@@ -1,7 +1,17 @@
 class SunSensor:
-    def __init__(self, lux) -> None:
-        self.__lux = lux
+    def __init__(self, x, y, z) -> None:
```

Comment:
I would like each sensor to be independent of the others with a lux value that returns if you could find a way to do this